robocopkaka/bootcamp-29-project
# Feat create admin profile

#### What does this PR do?
Create an AdminProfile component
#### Description of Task to be completed?
* Write tests for the component
* Create the component
* Add tabs to the AdminProfile component
* Add CSS for tabs to `/template/css/index.css`
* Create a utilityActions file used to setting a componentName in state
* Create a utilityActions reducer
#### How should this be manually tested?
* An admin who is logged in should be able to navigate to their profile and see all the events and centers currently in the system.
* They should be able to edit any center or event 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?(if applicable)
#155485307
